src/includes/performance/distributed-tracing/dotnet.mdx

```diff
@@ -0,0 +1,51 @@
+## Distributed Tracing
```

Comment:
this header should be under `custom-instrumentation.mdx ` otherwise it appears after the common content